circleguard/utils.py

```diff
@@ -220,6 +220,42 @@ def hitradius(CS):
     return np.float32(64 * ((1.0 - np.float32(0.7) * (float(np.float32(CS)) - 5) / 5)) / 2) * np.float32(1.00041)
 
 
+def GetQuartiles(arr):
```

Comment:
use snake case for function and variable names please https://peps.python.org/pep-0008/#function-and-variable-names